### DISCUSSION TITLE
GUI for Nomad Network

### DISCUSSION THREAD

**Discussion by hardBSDk** [markqvist/Reticulum#132] [Requests for New Features]
A Qt/GTK GUI for Nomad, as these toolkits allow easy software port for operating systems/graphical systems (Wayland).

For Android, it could be React or some lightweight GUI lib.

**Comment by markqvist** [comment]
I had plans for this when starting the code for nomadnet, and I am still thinking of adding it some time, but realistically it will probably be a little while before I have time to work on that.

Even though I really agree a GTK interface for nomadnet would be sweet!
